olive/systems/azureml/aml_evaluation_runner.py

```diff
@@ -49,6 +51,8 @@ def create_metric(metric_config, metric_args):
 
 
 def main(raw_args=None):
+    log_level = os.environ.get("OLIVE_LOG_LEVEL", "INFO")
```

Comment:
Also, maybe we can abstract this to lines into `olive.logging` as something like `set_verbosity_from_env`?

Reply:
done